lib/delta_filters.rb

```diff
@@ -0,0 +1,23 @@
+module Inbox
+  class DeltaFilters
+
+    attr_reader :filter_map
+
+    def self.build_exclude_types(object_types)
+      new.build_exclude_types(object_types)
+    end
+
+    def initialize(filter_map: API::OBJECTS_TABLE)
```

Comment:
Well, we still need to be compatible with Ruby 1.9.x. We still have customers using it.